deploy/role.yaml

```
@@ -115,6 +115,7 @@ rules:
   resources:
   - '*'
```

As we commented by slack, can we update it with specific permission? (Although has nothing to do with this PR)
```suggestion
  - 'autossls'
```

---

As discussed over slack, we will use wildcard `*` to avoid permission issues. For example, the `kind/status` is also required: `autossls/status, backends/status, zyncs/status`. Solved in https://github.com/3scale/saas-operator/pull/5/commits/a63e7ccded37fc5a5c6844825c9223e438e8407e